specs/protocol/isthmus/predeploys.md

```diff
@@ -192,6 +209,46 @@ The following functions are updated to read from the `L1Block` contract by calli
 - `otherBridge()(address)`
 - `OTHER_BRIDGE()(address)`
 
+### GasPriceOracle
+
+In order to maintain accurate offchain fee estimation, the `GasPriceOracle` must be updated to allow users
+to estimate the operator fee. We also add a new boolean `isIsthmus` to help with evaluating the operator fee.
+
+#### Interface
+
+##### `setIsthmus`
+
+This function is meant to be called once on the activation block of the isthmus network upgrade.
+It MUST only be callable by the `DEPOSITOR_ACCOUNT` once. When it is called, it MUST call
+call each getter for the network specific config and set the returndata into storage.
+
+```solidity
+function setIsthmus() external;
+```
+
+##### `getOperatorFee`
+
+This function calculates the operator fee based on the expected amount of gas used for a certain transaction.
+
+It uses the following values:
+
+- `operatorFeeScalar`
+- `operatorFeeConstant`
+- `isIsthmus`
+
+`operatorFeeScalar` and `operatorFeeConstant` are read from the `L1Block` contract, and `isIsthmus`
+is read directly from storage. If `isIsthmus` is false, then this function MUST return `0`.
+
+```function
+function getOperatorFee(uint256 gasUsed)(uint256)
+```
+
+##### `getL1Fee`
```

Comment:
If a user wanted to calculate the L1 data fee and the Operator fee separately as this is specified, then...

`l1DataFee=getL1Fee-getOperatorFee`
and `operatorFee=getOperatorFee`
which is strange. I can totally imagine a wallet accidentally thinking that the way to calculate the total non-L2 fee would be to do `getOperatorFee + getL1Fee`
While I also would like to keep as much backwards compatibility as possible, upon thinking about this more, I could imagine this being confusing, since the operatorFee doesn't actually have to have anything to do with L1 costs - for op-succinct it accounts for off-chain proving costs, which are completely separate from L1.

I think it makes more sense to keep getL1Fee as is (just L1 data fees) and have the separate getOperatorFee.

Then total fee is just getL1Fee + getOperatorFee